rel-eng/tito.props

```diff
@@ -985,6 +987,8 @@ disttag = .el7
 # don't whitelist foreman-release here, it's already in foreman-nightly-nonscl-rhel7
 whitelist =
   katello-host-tools
+  katello-pull-provider-migrate
+  python-beautifulsoup4
```

Comment:
what are these references to python-beautifulsoup4 ?  are they still needed?

Reply:
@jlsherrill must have been a rebase artifact.  Thought I did it right but guess not.  Removed.